benchmark/tangurnis/notes.txt

```diff
@@ -1 +1,14 @@
 This directory contains benchmarks defined in Tan, Gurnis '07 to check compressible convection
+
+
+
+Comparison of the heating terms for TALA, Di=0.5, gamma=1, a=0:
+
+                                       shear heating  adiabatic_heating
+
+analytical solution (Tan, Gurnis '07)  5.306916e-03   5.251999e-03
+
+numerical solution (refinement 4)      5.30690178e-03 5.25196977e-03
+numerical solution (refinement 5)      5.30691533e-03 5.25199721e-03
```

Comment:
This is just to test that the terms are implemented correctly. We compare the heating terms in the first time step (with time step length 0) to the analytical solution, so we don't really test the temperature solver, we just check if we have the correct RHS terms. But this way we can check that our approximations work exactly in the way other people have implemented them, and it's very easy to find where the problem is in case something changes in the future.